contracts/multisig-prover/src/state.rs

```diff
@@ -35,6 +36,7 @@ fn default_governance() -> Addr {
 
 pub const CONFIG: Item<Config> = Item::new("config");
 pub const COMMANDS_BATCH: Map<&BatchId, CommandBatch> = Map::new("command_batch");
+pub const MESSAGE_TO_SIGN: Map<&BatchId, MessageToSign> = Map::new("message_to_sign");
```

Comment:
This is only needed for the amplifier gateway right? If so, I would put this under the feature flag. Otherwise can be confusing.

Reply:
I am confused with this, too. What is this v.s. `COMMANDS_BATCH`?